src/_common/fireside/realm/realm.model.ts

```diff
@@ -0,0 +1,25 @@
+import { Model } from '../../model/model.service';
+import { Realm } from '../../realm/realm-model';
+
+export class FiresideRealm extends Model {
+	fireside_id!: number;
```

Comment:
we use declare now for models like this:
```
declare fireside_id: number;
```

It tells typescript of the field without actually outputting anything. The syntax we have right now technically would output a defineAttribute for the field with typescript, but I have disabled that funcionality while we migrate over.